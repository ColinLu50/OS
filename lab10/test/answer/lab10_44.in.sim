11487
603
627
473
597
449
