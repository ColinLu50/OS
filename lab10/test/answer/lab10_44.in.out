FCFS
160 -> 467 -> 290 -> 449 -> 281 -> 274 -> 439 -> 51 -> 449 -> 359 -> 101 -> 266 -> 61 -> 348 -> 357 -> 339 -> 98 -> 106 -> 326 -> 204 -> 89 -> 338 -> 444 -> 261 -> 259 -> 392 -> 210 -> 146 -> 249 -> 196 -> 128 -> 261 -> 15 -> 27 -> 441 -> 304 -> 291 -> 465 -> 125 -> 305 -> 126 -> 321 -> 339 -> 388 -> 276 -> 109 -> 163 -> 96 -> 123 -> 218 -> 203 -> 382 -> 305 -> 379 -> 287 -> 450 -> 352 -> 437 -> 238 -> 289 -> 355 -> 333 -> 351 -> 270 -> 339 -> 205 -> 115 -> 200 -> 223 -> 257 -> 383 -> 187 -> 256 -> 321 -> 184 -> 145 -> 106 -> 415 -> 395 -> 209 -> 79 -> 43 -> 332 -> 421 -> 130 -> 327 -> 406 -> 133
11487
SSTF
160 -> 163 -> 146 -> 145 -> 133 -> 130 -> 128 -> 126 -> 125 -> 123 -> 115 -> 109 -> 106 -> 106 -> 101 -> 98 -> 96 -> 89 -> 79 -> 61 -> 51 -> 43 -> 27 -> 15 -> 184 -> 187 -> 196 -> 200 -> 203 -> 204 -> 205 -> 209 -> 210 -> 218 -> 223 -> 238 -> 249 -> 256 -> 257 -> 259 -> 261 -> 261 -> 266 -> 270 -> 274 -> 276 -> 281 -> 287 -> 289 -> 290 -> 291 -> 304 -> 305 -> 305 -> 321 -> 321 -> 326 -> 327 -> 332 -> 333 -> 338 -> 339 -> 339 -> 339 -> 348 -> 351 -> 352 -> 355 -> 357 -> 359 -> 379 -> 382 -> 383 -> 388 -> 392 -> 395 -> 406 -> 415 -> 421 -> 437 -> 439 -> 441 -> 444 -> 449 -> 449 -> 450 -> 465 -> 467
603
SCAN
160 -> 146 -> 145 -> 133 -> 130 -> 128 -> 126 -> 125 -> 123 -> 115 -> 109 -> 106 -> 101 -> 98 -> 96 -> 89 -> 79 -> 61 -> 51 -> 43 -> 27 -> 15 -> 0 -> 163 -> 184 -> 187 -> 196 -> 200 -> 203 -> 204 -> 205 -> 209 -> 210 -> 218 -> 223 -> 238 -> 249 -> 256 -> 257 -> 259 -> 261 -> 266 -> 270 -> 274 -> 276 -> 281 -> 287 -> 289 -> 290 -> 291 -> 304 -> 305 -> 321 -> 326 -> 327 -> 332 -> 333 -> 338 -> 339 -> 348 -> 351 -> 352 -> 355 -> 357 -> 359 -> 379 -> 382 -> 383 -> 388 -> 392 -> 395 -> 406 -> 415 -> 421 -> 437 -> 439 -> 441 -> 444 -> 449 -> 450 -> 465 -> 467
627
C-SCAN
160 -> 146 -> 145 -> 133 -> 130 -> 128 -> 126 -> 125 -> 123 -> 115 -> 109 -> 106 -> 101 -> 98 -> 96 -> 89 -> 79 -> 61 -> 51 -> 43 -> 27 -> 15 -> 0 -> 476 -> 467 -> 465 -> 450 -> 449 -> 444 -> 441 -> 439 -> 437 -> 421 -> 415 -> 406 -> 395 -> 392 -> 388 -> 383 -> 382 -> 379 -> 359 -> 357 -> 355 -> 352 -> 351 -> 348 -> 339 -> 338 -> 333 -> 332 -> 327 -> 326 -> 321 -> 305 -> 304 -> 291 -> 290 -> 289 -> 287 -> 281 -> 276 -> 274 -> 270 -> 266 -> 261 -> 259 -> 257 -> 256 -> 249 -> 238 -> 223 -> 218 -> 210 -> 209 -> 205 -> 204 -> 203 -> 200 -> 196 -> 187 -> 184 -> 163
473
LOOK
160 -> 146 -> 145 -> 133 -> 130 -> 128 -> 126 -> 125 -> 123 -> 115 -> 109 -> 106 -> 101 -> 98 -> 96 -> 89 -> 79 -> 61 -> 51 -> 43 -> 27 -> 15 -> 163 -> 184 -> 187 -> 196 -> 200 -> 203 -> 204 -> 205 -> 209 -> 210 -> 218 -> 223 -> 238 -> 249 -> 256 -> 257 -> 259 -> 261 -> 266 -> 270 -> 274 -> 276 -> 281 -> 287 -> 289 -> 290 -> 291 -> 304 -> 305 -> 321 -> 326 -> 327 -> 332 -> 333 -> 338 -> 339 -> 348 -> 351 -> 352 -> 355 -> 357 -> 359 -> 379 -> 382 -> 383 -> 388 -> 392 -> 395 -> 406 -> 415 -> 421 -> 437 -> 439 -> 441 -> 444 -> 449 -> 450 -> 465 -> 467
597
C-LOOK
160 -> 146 -> 145 -> 133 -> 130 -> 128 -> 126 -> 125 -> 123 -> 115 -> 109 -> 106 -> 101 -> 98 -> 96 -> 89 -> 79 -> 61 -> 51 -> 43 -> 27 -> 15 -> 467 -> 465 -> 450 -> 449 -> 444 -> 441 -> 439 -> 437 -> 421 -> 415 -> 406 -> 395 -> 392 -> 388 -> 383 -> 382 -> 379 -> 359 -> 357 -> 355 -> 352 -> 351 -> 348 -> 339 -> 338 -> 333 -> 332 -> 327 -> 326 -> 321 -> 305 -> 304 -> 291 -> 290 -> 289 -> 287 -> 281 -> 276 -> 274 -> 270 -> 266 -> 261 -> 259 -> 257 -> 256 -> 249 -> 238 -> 223 -> 218 -> 210 -> 209 -> 205 -> 204 -> 203 -> 200 -> 196 -> 187 -> 184 -> 163
449
